app/src/main/java/com/nervousfish/nervousfish/modules/constants/IConstants.java

```diff
@@ -5,7 +5,8 @@
 import java.util.UUID;
 
 /**
- * Interface for a class containing all main constants of the application.
+ * Defines a class containing global constants for the application. These constants are definied in this file, because
```

Comment:
In IModule you omitted the use of "Defines a..." and here you introduce it, can you be more consistent?